src/pages/images/actions/forms/ImportImageFromTarballForm.tsx

```diff
@@ -0,0 +1,157 @@
+import { FC } from "react";
+import { useEventQueue } from "context/eventQueue";
+import { useFormik } from "formik";
+import { useToastNotification } from "context/toastNotificationProvider";
+import {
+  createImageAlias,
+  createImageFromInstance,
+  createImageFromUnifiedTB,
+} from "api/images";
+import {
+  ActionButton,
+  Button,
+  Form,
+  Input,
+  Modal,
+} from "@canonical/react-components";
+import * as Yup from "yup";
+import { Link } from "react-router-dom";
+
+interface Props {
+  close: () => void;
+  project: string;
+}
+
+const ImportImageFromTarballForm: FC<Props> = ({ close, project }) => {
+  const eventQueue = useEventQueue();
+  const toastNotify = useToastNotification();
+
+  //   const notifySuccess = () => {
```

Comment:
something happened in this file, seems like a bunch of code got commented out?

Reply:
This isn't supposed to be in this PR. I'm working on it. Youre so fast :joy: